Dockerfile

```diff
@@ -32,15 +32,14 @@ RUN pip3 install --no-cache-dir torch torchvision torchaudio --index-url https:/
 # Install runpod
 RUN pip3 install runpod requests
 
-# Download checkpoints/vae/LoRA to include in image
-RUN wget -O models/checkpoints/sd_xl_base_1.0.safetensors https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0/resolve/main/sd_xl_base_1.0.safetensors
-RUN wget -O models/vae/sdxl_vae.safetensors https://huggingface.co/stabilityai/sdxl-vae/resolve/main/sdxl_vae.safetensors
-RUN wget -O models/vae/sdxl-vae-fp16-fix.safetensors https://huggingface.co/madebyollin/sdxl-vae-fp16-fix/resolve/main/sdxl_vae.safetensors
-RUN wget -O models/loras/xl_more_art-full_v1.safetensors https://civitai.com/api/download/models/152309
-
-# Example for adding specific models into image
-# ADD models/checkpoints/sd_xl_base_1.0.safetensors models/checkpoints/
-# ADD models/vae/sdxl_vae.safetensors models/vae/
+ARG SKIP_DEFAULT_MODELS
```

Comment:
I've tested building with and without this flag, but I haven't tested the resulting container with the default models because uploading to dockerhub is exceptionally slow.

Technically we do not need this feature, adding src/extra_model_paths.yaml is enough to allow a user to add their own models

Reply:
Yes makes total sense. We will update our build-pipeline to provide an image that doesn't contain any other models, see https://github.com/blib-la/runpod-worker-comfy/issues/23